cube_helper/logger.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import sys
+
+
+def log_module():
+    logger = logging.getLogger(__name__)
+    if not getattr(logger, 'handler_set', None):
+        logger.setLevel(logging.INFO)
+        handler = logging.StreamHandler(sys.stdout)
```

Comment:
Hello John, I've gone over this and as I thought the stderr output is copied to the stdout so stderr still gets the error message it's supposed to. Should I write a test for this just to be sure?

Reply:
Yeah, it is quite hard to test because on the terminal stderr and stdout both show in the terminal. A unit test would be amazing to give reassurance that everything's going to the correct place. Can you wrote a stderr equivalent to `_redirect_stdout()` in `tests/common.py` for this testing?